framework/include/base/FEProblemBase.h

```diff
@@ -1197,6 +1197,10 @@ class FEProblemBase : public SubProblem, public Restartable
     _error_on_jacobian_nonzero_reallocation = state;
   }
 
+  bool igoreZerosJacobian() { return _igore_zeros_jacobian; }
+
+  void setIgoreZerosJacobian(bool state) { _igore_zeros_jacobian = state; }
+
```

Comment:
It is a parameter in `FEProblemBase`, so you would need something like this in your input file:
```
[Problem]
  type = FEProblem
  ignore_zeros_jacobian = true   # after fdkong fixes spelling of ignore
[]
```